docs/guides/server/db.adoc

```diff
@@ -244,7 +245,7 @@ A minimal Dockerfile to build an image which can be used with the {project_name}
 [source,dockerfile,subs="attributes+"]
 ----
 FROM quay.io/keycloak/keycloak:{containerlabel}
-ADD --chown=keycloak:keycloak https://github.com/awslabs/aws-advanced-jdbc-wrapper/releases/download/${properties["aws-jdbc-wrapper.version"]}/aws-advanced-jdbc-wrapper-${properties["aws-jdbc-wrapper.version"]}.jar /opt/keycloak/providers/aws-advanced-jdbc-wrapper.jar
+ADD --chmod=0666 https://github.com/awslabs/aws-advanced-jdbc-wrapper/releases/download/${properties["aws-jdbc-wrapper.version"]}/aws-advanced-jdbc-wrapper-${properties["aws-jdbc-wrapper.version"]}.jar /opt/keycloak/providers/aws-advanced-jdbc-wrapper.jar
```

Comment:
Not sure why this change is needed. But with the `--chown=keycloak:keycloak` (this is used for other drivers like [here](https://github.com/keycloak/keycloak/blob/d4b5e409a93dd3037d87e3f99e75c588c8154066/docs/guides/server/db.adoc?plain=1#L62)) I was getting the following error:
```
java.lang.reflect.InvocationTargetException                                                                                       │
│     at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)                                             │
│     at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)                           │
│     at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)                   │
│     at java.base/java.lang.reflect.Method.invoke(Method.java:568)                                                                 │
│     at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doReaugment(QuarkusEntryPoint.java:90)                                       │
│     at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:49)                                             │
│     at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:33)                                              │
│     at org.keycloak.quarkus.runtime.cli.command.Build.run(Build.java:85)                                                          │
│     at picocli.CommandLine.executeUserObject(CommandLine.java:2026)                                                               │
│     at picocli.CommandLine.access$1500(CommandLine.java:148)                                                                      │
│     at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)                         │
│     at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)                                                                  │
│     at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)                                                                  │
│     at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)                                              │
│     at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)                                                                 │
│     at picocli.CommandLine.execute(CommandLine.java:2170)                                                                         │
│     at org.keycloak.quarkus.runtime.cli.Picocli.runReAugmentation(Picocli.java:227)                                               │
│     at org.keycloak.quarkus.runtime.cli.Picocli.runReAugmentationIfNeeded(Picocli.java:159)                                       │
│     at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:110)                                                     │
│     at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:107)                                                      │
│     at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)                                             │
│     at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)                           │
│     at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)                   │
│     at java.base/java.lang.reflect.Method.invoke(Method.java:568)                                                                 │
│     at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:62)                                             │
│     at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:33)                                              │
│ Caused by: java.io.UncheckedIOException: java.io.IOException: Failed to create a new filesystem for /opt/keycloak/lib/../provider │
│ s/aws-advanced-jdbc-wrapper-2.3.3.jar                                                                                             │
│     at io.quarkus.paths.SharedArchivePathTree.open(SharedArchivePathTree.java:63)                                                 │
│     at io.quarkus.bootstrap.classloading.PathTreeClassPathElement.<init>(PathTreeClassPathElement.java:47)                        │
│     at io.quarkus.bootstrap.classloading.PathTreeClassPathElement.<init>(PathTreeClassPathElement.java:43)                        │
│     at io.quarkus.bootstrap.classloading.ClassPathElement.fromPath(ClassPathElement.java:83)                                      │
│     at io.quarkus.bootstrap.app.CuratedApplication.createDeploymentClassLoader(CuratedApplication.java:334)                       │
│     at io.quarkus.runner.bootstrap.AugmentActionImpl.createProductionApplication(AugmentActionImpl.java:174)                      │
│     at io.quarkus.deployment.mutability.ReaugmentTask.main(ReaugmentTask.java:69)                                                 │
│     ... 26 more                                                                                                                   │
│ Caused by: java.io.IOException: Failed to create a new filesystem for /opt/keycloak/lib/../providers/aws-advanced-jdbc-wrapper-2. │
│ 3.3.jar                                                                                                                           │
│     at io.quarkus.fs.util.ZipUtils.newFileSystem(ZipUtils.java:231)                                                               │
│     at io.quarkus.fs.util.ZipUtils.newFileSystem(ZipUtils.java:210)                                                               │
│     at io.quarkus.paths.ArchivePathTree.openFs(ArchivePathTree.java:179)                                                          │
│     at io.quarkus.paths.SharedArchivePathTree.open(SharedArchivePathTree.java:61)                                                 │
│     ... 32 more                                                                                                                   │
│ Caused by: java.nio.file.AccessDeniedException: /opt/keycloak/lib/../providers/aws-advanced-jdbc-wrapper-2.3.3.jar                │
│     at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:90)                                           │
│     at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)                                            │
│     at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)                                            │
│     at java.base/sun.nio.fs.UnixFileSystemProvider.checkAccess(UnixFileSystemProvider.java:313)                                   │
│     at jdk.zipfs/jdk.nio.zipfs.ZipFileSystem.<init>(ZipFileSystem.java:170)                                                       │
│     at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.getZipFileSystem(ZipFileSystemProvider.java:125)                             │
│     at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.newFileSystem(ZipFileSystemProvider.java:120)                                │
│     at io.quarkus.fs.util.ZipUtils.newFileSystem(ZipUtils.java:228)                                                               │
│     ... 35 more                                                                             
```

@keycloak/cloud-native Any ideas?